aws-java-sdk-core/src/test/java/com/amazonaws/util/StringUtilsTest.java

```diff
@@ -100,7 +230,7 @@ public void lowerCase_NullString() {
 
     @Test
     public void lowerCase_EmptyString() {
-        Assert.assertThat(StringUtils.lowerCase(""), Matchers.isEmptyString());
+        assertThat(StringUtils.lowerCase(""), Matchers.isEmptyString());
```

Comment:
Again, keep these as they are & just use Assert.whatever for the tests we've added.

Reply:
Reverted changes